### ISSUE TITLE
keeps crashing

### ISSUE THREAD

**Issue by LadyJaneBrown** [rmcrackan/Libation#299]
Hi I love the idea of the program but everytime I try and download a book. it closes. please find attached my log fil
[Log202206_002.log](https://github.com/rmcrackan/Libation/files/8985715/Log202206_002.log)
e

**Comment by Mbucari** [comment]
Let me guess, you are set up to download your books as MP3s, split into chapters? If so, we've found the bug and just waiting on @rmcrackan to accept some changes and publish a release. Sorry, this was a really difficult bug to find!

**Comment by rmcrackan** [comment]
Duplicate of #294

**Comment by rmcrackan** [comment]
I'm so sorry about this bug. A fix is going out very soon. In the meantime, I'm tracking this issue in #294
